charts/argocd-image-updater/Chart.yaml

```
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Mount ssh-known-hosts from argocd"
+    - "[Changed]: Bump image tag to v0.11.2"
```

We only need the latest change in here. The old change can be dropped/replaced

```suggestion
    - "[Changed]: Update to app version v0.11.2"
```